nbsphinx_link/__init__.py

```diff
@@ -58,7 +58,7 @@ def copy_file(src, dest, document):
     src : str
         Path to the source file.
     dest : str
-        Path to the destination file.
+        Path to the destination file or directory.
```

Comment:
``copy_file`` does only copy a single file, since ``shutil.copy`` does only copy a single file.

Reply:
Yes, but the *destination* can be either a full file path, or a directory path.

Reply:
Oh yes, true that.